tests/vcr_python_wrapper.py

```diff
@@ -107,11 +107,44 @@ def filter_response(response):
         # headers should be case insensitive, but for some reason they weren't for me
         response['headers'].pop(header.lower(), None)
         response['headers'].pop(header, None)
+    try:
+        json_body = json.loads(response['body']['string'])
+    except json.JSONDecodeError:
+        pass
+    else:
+        if "host" in json_body:
+            json_body['host'] = 'FILTERED'
+        if "account" in json_body:
+            json_body['account'] = 'FILTERED'
+        if "base_dn" in json_body:
+            json_body['base_dn'] = 'FILTERED'
+        if "groups_base" in json_body:
+            json_body['groups_base'] = 'FILTERED'
+        # another for the search result
+        if 'results' in json_body:
+            if "host" in json_body['results'][0]:
```

Comment:
It seems this line is failing if the results array is empty.
Maybe you should filter this only if `length(json_body["results"]) ==1` or you should filter every item in results.

@evgeni thoughts?